app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapControls.jsx

```diff
@@ -414,7 +416,38 @@ export default class SamplesHeatmapControls extends React.Component {
     return filterTags;
   };
 
+  renderFilterStatsInfo = () => {
+    let { filteredTaxaCount, totalTaxaCount } = this.props;
+    return (
+      <span className={cs.reportInfoMsg}>
+        Showing {filteredTaxaCount} taxa of {totalTaxaCount} preselected taxa.
+        <ColumnHeaderTooltip
+          trigger={
+            <span>
+              <InfoSIcon className={cs.infoIcon} />
+            </span>
+          }
+          content="The data included in this heatmap 
+            was preselected based on the following conditions:"
+          list={[
+            "The top 1,000 unique taxa per sample",
```

Comment:
how is the top 1,000 computed? I would assume as a user that it is ranked by the current selected metric, which I don't believe is our intention.

Reply:
It's based off the rPM! I updated the tooltip's copy, here's the new screenshot.
![image](https://user-images.githubusercontent.com/53838890/77003152-08725c00-691a-11ea-87fb-a5ff4f863d67.png)

Reply:
Nice! I'd shorten it to "top 1000 unique taxa per sample by rPM" but I will leave the final copy to you and @happyimadesignr .